VirtualAssistantBusinessLogic/VirtualAssistantBusinessLogic/SparQL/SparQLWhere.cs

```diff
@@ -23,31 +23,31 @@ public SparQLWhere(SparQLSelect sparQLSelect)
                 EncodedSPOs.Add(SparQLSelect.FromSubjectRaw, SparQLSelect.FromSubject);
             }
         }
-        private string SubjectString { get; set; } = "";
-        private string ObjectString { get; set; } = "";
-        private string PredicateString { get; set; } = "";
-        private List<string> Conditions { get; set; }
+        public string SubjectString { get; private set; } = "";
+        public string ObjectString { get; private set; } = "";
+        public string PredicateString { get; private set; } = "";
+        public List<string> Conditions { get; private set; }
         private const string labelServiceSparQL = "SERVICE wikibase:label { bd:serviceParam wikibase:language \"en\". }";
         public SparQLSelect SparQLSelect;
-        private Dictionary<string, EncodedSPO> EncodedSPOs { get; set; }
+        public Dictionary<string, EncodedSPO> EncodedSPOs { get; private set; }
 
         /// <summary>
         /// Specifies the subject of the sparql triplet as being equal to the
         /// input subject.
         /// </summary>
         /// <param name="subject">Subject of the sparql triplet</param>
-        /// <returns></returns>
+        /// <returns>this</returns>
```

Comment:
Write out what is meant with this, guessing its the subject?

Reply:
Enig...noget ala "This object"

Reply:
```suggestion
        /// <returns>this class instance</returns>
```